app/serializers/my_request/request/show_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class MyRequest::Request::ShowSerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :reward, :offers
+
+  def offers
+    offer_uids = []
+    object.offers.each do |offer|
+      helper = User.find(offer.helper_id)
+      offer_uids << helper.uid
+    end
```

Comment:
Could do:
```
object.offers.map do |offer| 
  User.find(offer.helper_id).uid 
end
```

Reply:
Also, it would perhaps be good to return a list of objects, not just a list of strings, like:
`offers: [
  { email: helper@mail.com },
  {...}
]`
As this will allow us to add stuff like user ratings inside the object without having to redo the logic in the frontend.

Reply:
I tried that before, but it wouldn't for some reason...
I did change the response to include the objects with `:email`.